packages/driver/src/multi-domain/index.ts

```diff
@@ -63,6 +63,11 @@ const setup = () => {
     addLog () {},
   }
 
+  // @ts-ignore
```

Comment:
Ignoring the cy type for now because i think this cy object isn't fully filled out yet.